source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs

```diff
@@ -127,5 +157,21 @@ public static bool PerformSetup(string appId, string nearbySvcId)
 
             return true;
         }
+		
```

Comment:
Can this be merged into GPGSUtil.UpdateGameInfo so it is being updated only from one method?